pkg/solana/txm/txm.go

```diff
@@ -88,10 +94,25 @@ func NewTxm(chainID string, tc func() (client.ReaderWriter, error), cfg config.C
 		cfg:    cfg,
 		txs:    newPendingTxContextWithProm(chainID),
 		ks:     ks,
-		client: utils.NewLazyLoad(tc),
+		client: client,
+		sendTx: sendTx,
 	}
 }
 
+// SendTx sends a transaction using a single client or an override sendTx function
+func (txm *Txm) SendTx(ctx context.Context, tx *solanaGo.Transaction) (solanaGo.Signature, error) {
+	if txm.sendTx != nil {
+		return txm.sendTx(ctx, tx)
+	}
+
+	// Send tx using a single RPC client
+	client, err := txm.client.Get()
+	if err != nil {
+		return solanaGo.Signature{}, err
+	}
+	return client.SendTx(ctx, tx)
+}
```

Comment:
Maybe it would make more sense to pass the `TransactionSender` directly rather than an override function? My thought was to make it generic, but it might be more clear to just use the component itself. 

I don't see a nice way to wrap it since `client.Get()` is used for returning a ReaderWriter which is essentially a single RPC, while using the `TransactionSender` is a layer higher using all available RPCs.